File: alumni-back/src/main/java/ec/edu/ista/springgc1/model/dto/QuestionWithAnswersDTO.java
package ec.edu.ista.springgc1.model.dto;

import java.io.Serializable;
import java.util.List;

import com.fasterxml.jackson.annotation.JsonView;

import ec.edu.ista.springgc1.view.View;
import lombok.Data;

@Data
public class QuestionWithAnswersDTO implements Serializable {
	@JsonView(View.Public.class)
    private Long questionId;
	@JsonView(View.Public.class)
    private String surveytitle;
	@JsonView(View.Public.class)
    private String suverydescricpion;
	@JsonView(View.Public.class)
    private String questionText;
	@JsonView(View.Public.class)
    private List<String> answers;
}

File: alumni_app/src/app/data/model/DTO/QuestionWithAnswersDTO.ts
export class QuestionWithAnswersDTO {
    questionId: number;
    surveyTitle: string;
    surveyDescription: string;
    questionText: string;
    answers: string[];
  
    constructor(
      questionId: number,
      surveyTitle: string,
      surveyDescription: string,
      questionText: string,
      answers: string[]
    ) {
      this.questionId = questionId;
      this.surveyTitle = surveyTitle;
      this.surveyDescription = surveyDescription;
      this.questionText = questionText;
      this.answers = answers;
    }
  }